scripts/jenkins/track-upstream-and-package.pl

```diff
@@ -283,6 +283,9 @@ ()
     eval {
       $custom_service=xml_get_text($xmldom, '/services/service[@name="download_files"][1]/param[@name="changesgenerate"][1]');
     };
+    eval {
+      $custom_service=xml_get_text($xmldom, '/services/service[@name="renderspec"][1]/param[@name="output-name"][1]');
```

Comment:
Is it correct to assume that `renderspec` is not present together with `download_files` or `github_tarballs` `git_tarballs`?
Because they all assign the same variable. So only the last one wins if there are any two of the entries present. Not sure if this is wanted or only one can be present at a time.